### PR TITLE
Convert to dark mode

### PR DESCRIPTION
**Pull request by deepesh-dg**

### CODE REVIEW

**Comment by wajeshubham** [comment]
Hey @deepesh-dg hamburger menu is pending to be converted to dark mode. Here's the image for a reference 

Thanks

File: src/utils/classNames.ts
Comment (wajeshubham) [inline src/utils/classNames.ts:1]: No need to create separate utils files for each function like `classNames` and `renderHTMLFromJSX`. Keep both the functions in the `utils/index.ts file`

**Review by wajeshubham — COMMENTED** [review]
@deepesh-dg Kindly look into the code review. Thanks!

File: src/utils/classNames.ts
Comment (deepesh-dg) [reply src/utils/classNames.ts:1]: Done

**Review by wajeshubham — APPROVED** [review]

**Pull request merged**